compiler/passes/normalize.cpp

```diff
@@ -887,6 +889,54 @@ static void processSyntacticDistributions(CallExpr* call) {
   }
 }
 
+/* Find patterns like
+     (new (call manager (call ClassType init-args)))
+     ... where manager might be _owned _to_raw _shared
```

Comment:
It's a bit odd that you use "_to" prefix for unmanaged only. Worth a comment somewhere as to this choice.